pkg/services/audit/auditor/por.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/hex"
 	"sync"
 
+	"github.com/nspcc-dev/neofs-node/pkg/innerring/processors/audit"
```

Comment:
I don't like this dependency, can we move `SGSource` together with it's parameters to `pkg/core/storagegroup` package, similar to where `netmap.Source` and `container.Source` are defined?

Reply:
moved

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1402/commits/4c84c0fe541d6c5f58502ea85b3f3bab0bfecd63